R/pool.R

```diff
@@ -186,7 +259,11 @@ Pool <- R6Class("Pool",
           NULL
         )
         if (!is.null(removeFrom)) {
-          rm(list = id, envir = removeFrom)
```

Comment:
Yeah, that makes everything a lot simpler and more readable :) I'll just deal with the merge conflicts, it's fine